renderers/lit/src/0.8/ui/component-registry.ts

```diff
@@ -17,13 +17,15 @@
 import { CustomElementConstructorOf } from "./ui.js";
 
 export class ComponentRegistry {
+  private schemas: Map<string, unknown> = new Map();
   private registry: Map<string, CustomElementConstructorOf<HTMLElement>> =
     new Map();
 
   register(
     typeName: string,
     constructor: CustomElementConstructorOf<HTMLElement>,
-    tagName?: string
+    tagName?: string,
+    schema?: unknown
```

Comment:
We don't have a better definition for this than `unknown`? Looking at the example farther down these diffs, it does look like we could provide a better type than this.

Reply:
done